_posts/2018-10-01-project-roles.md

```diff
@@ -0,0 +1,81 @@
+
+### Responding to ad-hoc situations
+
+If project is on fire, you’re the first line of defence. You may (and probably need to) delegate it further, but you’re the rapid response team. This allows an easy way out for every team member if something goes wrong and an urgent fix is required, for example the project needs an additional developer, because the project is behind schedule. Without this role, the responsibility would be distributed among the whole team and as such it would be ineffective.
+
+### Reporting
+
+You can present project status to stakeholders and bring anyone up to date. This may relate only to certain fields of the project, such as development, design or other.
+
+### Documenting
+
+Updating knowledge on Confluence, so everyone is on the same page. Documenting has a broader meaning, but for the purpose of defining roles, let’s just stick to gathering all knowledge and arrangements in one place. This may be as trivial as meeting notes — someone has to do it. Without this role, there wouldn’t be a single source of truth about the project; all would be scattered between JIRA tasks, emails, bug trackers, slack, etc.
+
+### Risk Management
+
+This is both bringing awareness of project risks to the team, as well as acting on the current identified risks. For example, if we have a tight schedule, we need to pay extra attention to our estimates, and regular plannings should reflect that, and the team need to be aware of the priorities — no needless refactors this time, sorry. 
+
+### Project Management
+
+Managing everything related to JIRA and short term planning — daily standups, weekly or bi-weekly plannings, assigning tasks if that rocks your boat, putting down estimations, looking at burndowns if you have them.
+
+### Traffic / Role delegation
+
+Helping in deciding which team member is best for the task at hand. Assigning roles mentioned here is also a traffic job, so I guess this role needs to be assigned first. 
+
+### Planning / Task estimation
+
+Self explanatory. In contrast to project management, this is about giving the estimates during planning using your best knowledge and expertise. This also means estimating larger chunks of projects basing on your experience, not the actual detailed specs. This can guide high-level decisions as to what direction to choose, which parts of the project are unprofitable, etc.
+
+### Announcing goals
+
+Communicating to the team what is the most important at a given point in time. In general this can be understood as providing a transparent communication to the team.
```

Comment:
"providing a transparent communication to the team" - czy tak sie uzywa provide. Chyba "provide somebody with something".